libs/chotify/chotify/chotify.lua

```diff
@@ -39,14 +39,19 @@ function Chotify:Post(obj)
     local body = obj.body or ""
     local icon = obj.icon or ""
     local time = obj.time or 5
+    local sound = obj.sound or nil
```

Comment:
"or nil" makes no sense here. The point of "or X" is to provide default values, which sound doesn't need (yet).